tests/app/questionnaire/test_rules.py

```diff
@@ -106,6 +107,134 @@ def test_evaluate_goto_returns_true_when_value_contained_in_list(self):
 
         self.assertTrue(evaluate_goto(goto, {}, answer_store, 0))
 
+    def test_evaluate_skip_condition_returns_true_when_this_rule_true(self):
+        # Given
+        skip_condition = [{
+                'when': [
+                    {
+                        'id': 'this',
+                        'condition': 'equals',
+                        'value': 'value'
+                    }
+                ]
+            },
+            {
+                'when': [
+                    {
+                        'id': 'that',
+                        'condition': 'equals',
+                        'value': 'other value'
+                    }
+                ]
+            }
+        ]
+        answer_store = AnswerStore()
+        answer_store.add(Answer(answer_id='this', value='value'))
+
+        # When
+        condition = evaluate_skip_condition(skip_condition, {}, answer_store)
+
+        # Given
+        self.assertTrue(condition)
+
+    def test_evaluate_skip_condition_returns_true_when_that_rule_true(self):
+
+        skip_condition = [{
+                'when': [
+                    {
+                        'id': 'this',
+                        'condition': 'equals',
+                        'value': 'value'
+                    }
+                ]
+            },
+            {
+                'when': [
+                    {
+                        'id': 'that',
+                        'condition': 'equals',
+                        'value': 'other value'
+                    }
+                ]
+            }
+        ]
+        answer_store = AnswerStore()
+        answer_store.add(Answer(answer_id='that', value='other value'))
+
+        self.assertTrue(evaluate_skip_condition(skip_condition, {}, answer_store))
+
+    def test_evaluate_skip_condition_returns_true_when_both_or_rules_true(self):
```

Comment:
Should the title of this include 'both' or 'all' since we can have more than 2 or conditions?